src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -578,6 +633,20 @@ public void user_should_not_complete_task_when_grant_type_specific_and_permissio
             .body("status", equalTo(403))
             .body("detail", equalTo(ROLE_ASSIGNMENT_VERIFICATION_DETAIL_REQUEST_FAILED));
 
+        String serviceToken = caseworkerCredentials.getHeaders().getValue(AUTHORIZATION);
+        UserInfo userInfo = authorizationProvider.getUserInfo(serviceToken);
+        //This will be changed when assign endpoint story RWA-1582 is played
+        Map<String, Matcher<?>> taskValueMap = Map.of(
```

Comment:
Should we consider the deployment order if you're dependent on another ticket?  We don't want to have `null` values being set in the DB.  
When this feature is live every update to the DB will have an Action added and so there should be values populated here already.

Maybe a better solution here would be to check the state of the values before you run the test case and then verify that the values have not changed?  This way it is independent of other changes.

Reply:
true, currently when task is initially created they are null, since we have done only claim, unclaim, complete endpoints, only those actions are saved in the DB, but once all endpoints are addressed we can see the gaps which endpoints are not covered or which of the cases these attributes can be null, probably we can discuss in a call?